### ISSUE TITLE
Sparrow address reuse

### ISSUE THREAD

**Issue by RF311** [sparrowwallet/sparrow#1422]
Hello
I have reused a Sparrow wallet-generated address several times to allow automatic transactions to be made to my wallet, but the last transaction did not go through. When viewing the wallet in Sparrow, it shows a newly generated address. When I try to set the original address, it has a red border around it. The transition is not showing on the wallet but is showing several confirmations on mempool.
I am seeking some assistance, please. Thank you.

**Comment by RF311** [comment]
Sorry, I meant to write "Transaction," not "Transition".
I am now learning through the Bitcoin community to always create a fresh address to preserve privacy. But this is a hard pill to swallow when you have a trusted address, and you don't see the transaction on your Sparrow Wallet. I don't have the Sparrow Wallet with me all the time, so I tend to reuse a trusted address to move the BTC off exchanges and other wallets.
I guess I am looking for assistance on why the transaction using the trusted address on Sparrow did not show up in my wallet.
If anyone could assist would be much appreciated.
Thank you

**Comment by RF311** [comment]
Can anyone help me understand why a trusted address can not be reused on a Sparrow Wallet after being reused several times? Is there an expiration date on the address? I am confused because some exchanges want you to use a whitelisted address, so it seems they encourage you to use the same address. 
Again, if anyone has some input on this, I would greatly appreciate it.
Thank you.

**Comment by RF311** [comment]
Hello, Thank you for your reply.
I tried refreshing the wallet under "View" and " Refreah Wallet," and the address bar reverted back to the address (.../0/1). When I am looking to receive on address (.../0/0), when I place address (.../0/0) in the address bar under the "Receiving" Tab, there is a red border around the address bar.

I have used this one address four other times, according to the UTXO tab, but the fifth one, which is this last one, is not logged or showing received by either Sparrow or Bitcoin Core Node. The status bar on Sparrow, in the lower right-hand corner, is green. When I initially sign in to my wallet on Sparrow and select the Receive Tab, the address shows a new address. When I select the Address Tab, the one address I have used several times is on top of the listed "Receive Addresses" (../0/0). The address below it (.../0/1) is the new address viewed on the Receiving Tab address bar. When I copy and paste the address (.../0/0) in place of the address (.../0/1) in the address bar under the Receive Tab, the address bar has a red border.
Also, under the UTXO tabs, where it shows the last four transactions, there is a red dot at the need of the transaction line, and when you hover over it with the mouse pointer, it states duplicate. I have not seen that before.

I tried refreshing the wallet under "View" and " Refreah Wallet," and the address bar reverted back to the address (.../0/1). I checked Mempool under the transaction ID, and it shows 1418 confirmations. I ran the address (.../0/0) under Mempool search, and it is showing an accurate count of what it should be in my wallet. 

I am just perplexed about why Sparrow and Bitcoin Core Node are not recording the transaction.  I have sent BTC to another wallet in Sparrow and Bitcoin Core Node, and Sparrow shows it was received with no issues. I am a little hesitant to do a test run on the new address (.../0/1) just for fear of losing the last address being used. I am not that tech savvy and probably over-paranoid. 

I am interested in knowing if you have encountered this issue before and can shed some light on this.

Thank you
